datasets/swda/README.md

```diff
@@ -87,118 +87,51 @@ Utterance are tagged with the [SWBD-DAMSL](https://web.stanford.edu/~jurafsky/ws
-Signal-non-understanding       |          br          |   298    |   0.15   |       286       |   0.15   |        9        |   0.22   |        3        |   0.09  
-Other Answers                  |          no          |   284    |   0.14   |       277       |   0.14   |        6        |   0.15   |        1        |   0.03  
-Conventional-opening           |          fp          |   225    |   0.11   |       220       |   0.11   |        5        |   0.12   |        0        |   0.00  
-Or-Clause                      |         qrr          |   209    |   0.10   |       206       |   0.11   |        2        |   0.05   |        1        |   0.03  
-Dispreferred Answers           |        arp_nd        |   207    |   0.10   |       204       |   0.11   |        3        |   0.07   |        0        |   0.00  
-3rd-party-talk                 |          t3          |   117    |   0.06   |       115       |   0.06   |        0        |   0.00   |        2        |   0.06  
-Offers, Options Commits        |       oo_co_cc       |   110    |   0.06   |       109       |   0.06   |        0        |   0.00   |        1        |   0.03  
-Maybe/Accept-part              |        aap_am        |   104    |   0.05   |       97        |   0.05   |        7        |   0.17   |        0        |   0.00  
-Downplayer                     |          t1          |   103    |   0.05   |       102       |   0.05   |        1        |   0.02   |        0        |   0.00  
-Self-talk                      |          bd          |   103    |   0.05   |       100       |   0.05   |        1        |   0.02   |        2        |   0.06  
-Tag-Question                   |          ^g          |    92    |   0.05   |       92        |   0.05   |        0        |   0.00   |        0        |   0.00  
-Declarative Wh-Question        |         qw^d         |    80    |   0.04   |       79        |   0.04   |        1        |   0.02   |        0        |   0.00  
-Apology                        |          fa          |    79    |   0.04   |       76        |   0.04   |        2        |   0.05   |        1        |   0.03  
-Thanking                       |          ft          |    78    |   0.04   |       67        |   0.03   |        7        |   0.17   |        4        |   0.12  
-
-
-![Label Frequencies](https://raw.githubusercontent.com/NathanDuran/Switchboard-Corpus/master/swda_data/metadata/Swda%20Label%20Frequency%20Distributions.png)
+
+'swda_filename':       (str) The filename: directory/basename
+
+'ptb_basename':        (str) The Treebank filename: add ".pos" for POS and ".mrg" for trees
+
+'conversation_no':     (int) The conversation Id, to key into the metadata database.
+
+'transcript_index':    (int) The line number of this item in the transcript (counting only utt lines).
+
+'act_tag':             (list of str) The Dialog Act Tags (separated by ||| in the file).
```

Comment:
Can you redirect to the list of tags with their meanings ?
Or maybe include them in the dataset card ? (it might be very long though)

Reply:
I added their meaning. I assume people would look at the official dataset link provided if they need more details 🤔